taskcluster/io_modules/decoders/taskcluster/live_backing_log.lua

```diff
@@ -117,7 +117,8 @@ local md_gh_eop     = l.P"/" + (".git" * l.P"/"^-1)
 local md_gh_project = l.C((l.P(1) - md_gh_eop)^1)
 local md_gh_api     = l.C("api.github.com") * "/repos/" * md_any * md_gh_project
 local md_gh         = l.C("github.com") * "/" * md_any * md_gh_project * md_gh_eop * (l.P"raw" * "/" * md_revision)^-1
-local md_source     = md_protocol * (md_hg + md_gh + md_gh_api + md_any^-4)
+local md_hgpush     = l.C("tools.taskcluster.net/hooks/hg-push") * "/" * l.Cc(nil) * md_any
```

Comment:
nit: I know this is called `hgpush` based on the url component, but maybe a hook-ish name would be clearer down the line.

Reply:
I am open to suggestions `md_hgpush_hook`?.  It is very localized (only used once immediately below) so at least all the context is in one place.